bootstrap.go

```diff
@@ -9,16 +9,22 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/profiler"
 )
 
+type ConnectionType int
+
+const (
+	SocketConnection ConnectionType = iota
```

Comment:
Also for readability and autocomplete it's better name it `ConnectionTypeSocket` `ConnectionTypeFoo` `ConnectionTypeBar`

Since in code you will call "coopdatadog.Connection...`

Reply:
I agree